fabric-rendering-v1/src/testmodClient/java/net/fabricmc/fabric/test/rendering/client/HudLayerTests.java

```diff
@@ -0,0 +1,147 @@
+	}
+
+	private static void renderAfterExperienceLevel(DrawContext context, RenderTickCounter tickCounter) {
+		if (!shouldRender) return;
+		// Render a green rectangle at the bottom of the screen, and it should block the hotbar and status bars
+		context.fill(context.getScaledWindowWidth() / 2 - 50, context.getScaledWindowHeight() - 50, context.getScaledWindowWidth() / 2 + 50, context.getScaledWindowHeight() - 10, Colors.GREEN);
+		context.drawCenteredTextWithShadow(MinecraftClient.getInstance().textRenderer, "3. This green rectangle should block the hotbar and status bars.", context.getScaledWindowWidth() / 2, context.getScaledWindowHeight() - 40, Colors.WHITE);
+	}
+
+	private static void renderBeforeDemoTimer(DrawContext context, RenderTickCounter tickCounter) {
+		if (!shouldRender) return;
+		// Render a yellow rectangle at the right of the screen, and it should be above the sleep overlay but below the scoreboard
+		context.fill(context.getScaledWindowWidth() - 240, context.getScaledWindowHeight() / 2 - 10, context.getScaledWindowWidth(), context.getScaledWindowHeight() / 2 + 10, Colors.YELLOW);
+		context.drawTextWithShadow(MinecraftClient.getInstance().textRenderer, "4. This yellow rectangle should be above", context.getScaledWindowWidth() - 236, context.getScaledWindowHeight() / 2 - 10, Colors.WHITE);
+		context.drawTextWithShadow(MinecraftClient.getInstance().textRenderer, "the sleep overlay but below the scoreboard.", context.getScaledWindowWidth() - 236, context.getScaledWindowHeight() / 2, Colors.WHITE);
+	}
+
+	private static void renderBeforeChat(DrawContext context, RenderTickCounter tickCounter) {
+		if (!shouldRender) return;
+		// Render a blue rectangle at the bottom left of the screen, and it should be blocked by the chat
+		context.fill(0, context.getScaledWindowHeight() - 40, 300, context.getScaledWindowHeight() - 50, Colors.BLUE);
+		context.drawTextWithShadow(MinecraftClient.getInstance().textRenderer, "5. This blue rectangle should be blocked by the chat.", 0, context.getScaledWindowHeight() - 50, Colors.WHITE);
+	}
+
+	private static void renderAfterSubtitles(DrawContext context, RenderTickCounter tickCounter) {
+		if (!shouldRender) return;
+		// Render a yellow rectangle at the top of the screen, and it should block the player list
+		context.fill(context.getScaledWindowWidth() / 2 - 150, 0, context.getScaledWindowWidth() / 2 + 150, 15, Colors.YELLOW);
+		context.drawCenteredTextWithShadow(MinecraftClient.getInstance().textRenderer, "6. This yellow rectangle should block the player list.", context.getScaledWindowWidth() / 2, 0, Colors.WHITE);
+	}
+
+	@Override
+	public void runTest(ClientGameTestContext context) {
+		// Set up required test environment
+		context.getInput().resizeWindow(1708, 960); // Twice the default dimensions
+		context.runOnClient(client -> {
+			client.options.hudHidden = false;
```

Comment:
This shouldn't be necessary as the client gametest runner resets all game options to the defaults before each test

Reply:
As discussed on Discord, the GUI scale was the problem, which makes sense because GUI scale 2 isn't the default.